patches/unix/node-gyp-build+4.2.3.patch

```diff
@@ -1,5 +1,5 @@
 diff --git a/node_modules/node-gyp-build/index.js b/node_modules/node-gyp-build/index.js
-index 3e39703..5e84906 100644
+index b5096ed..7cd451a 100644
 --- a/node_modules/node-gyp-build/index.js
 +++ b/node_modules/node-gyp-build/index.js
```

Comment:
Apparently `node-gyp-build` had been updated in some previous PR that got merged, without the patch file being updated. Updating it here -- incidental to this PR.